packages/adapter-squid/test/integration/SquidAdapter.integration.test.ts

```diff
@@ -0,0 +1,89 @@
+import "reflect-metadata";
+import * as path from "path";
+import pkg from "../../package.json";
+import { DependencyRegistrar } from "../../src/DependencyRegistrar";
+import {
+   ConfigProvider,
+   IConfigProvider,
+   DependencyRegistrar as SdkDependencyRegistrar,
+} from "@mate/sdk";
+import { SquidAdapter } from "../../src/SquidAdapter";
+import { container } from "tsyringe";
+import "dotenv/config";
+
+describe("SquidAdapter Integration", () => {
+   let adapter: SquidAdapter;
+   let configProvider: IConfigProvider;
+
+   beforeAll(() => {
+      // Override the config file path to use the example config
+      process.env["CONFIG_PATH"] = path.resolve(
+         __dirname,
+         "../../../../config.example.yaml"
```

Comment:
same idea here, replacing `__dirname` with `process.cwd()` might save you a few `../`